Uno.Gallery/Uno.Gallery.UWP/Views/SamplePages/LightSensorSamplePage.xaml.cs

```diff
@@ -0,0 +1,132 @@
+﻿using System;
+using System.ComponentModel;
+using Windows.Devices.Sensors;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.Gallery.Views.Samples
+{
+	[SamplePage(SampleCategory.NonUIFeatures, "Light Sensor", Description = "Represents an ambient light sensor. The sensor returns illuminance in LUX.", DocumentationLink = "https://learn.microsoft.com/en-us/windows/uwp/devices-sensors/use-the-light-sensor")]
+	public sealed partial class LightSensorSamplePage : Page
+	{
+		public LightSensorSamplePage()
+		{
+			this.InitializeComponent();
+		}
+
+		private void ObserveReadingChangeButton_Click(object sender, RoutedEventArgs e)
+		{
+			if ((sender as Button)?.DataContext is LightSensorSamplePageViewModel viewModel)
+			{
+				if (!viewModel.ObservingReadingChange)
+				{
+					viewModel.StartObserveReadingChange();
+				}
+				else
+				{
+					viewModel.StopObservingReadingChange();
+				}
+			}
+		}
+	}
+
+	public class LightSensorSamplePageViewModel : INotifyPropertyChanged
```

Comment:
There's also a ViewModelBase class [here](https://github.com/unoplatform/Uno.Gallery/blob/master/Uno.Gallery/Uno.Gallery.Shared/ViewModels/ViewModelBase.cs) that already inherits `INotifyPropertyChanged`

Not a big deal but just an FYI :)

Reply:
Thanks, did not know that. I've changed to the BaseClass which makes the sample-ViewModel a bit lighter. :)